stl/inc/xutility

```diff
@@ -5466,7 +5466,8 @@ _NODISCARD constexpr bool _Within_limits(const _Ty& _Val, false_type, false_type
 template <class _InIt, class _Ty>
 _NODISCARD constexpr bool _Within_limits(_InIt, const _Ty& _Val) { // check whether _Val is within the limits of _Elem
     using _Elem = remove_pointer_t<_InIt>;
-    return _Within_limits(_Val, is_signed<_Elem>{}, is_signed<_Ty>{}, bool_constant<-1 == static_cast<_Ty>(-1)>{});

```

Comment:
I don't like this change as much as the others, I think it impacts clarity more. I'm OK with it if the throughput improvement is actually decent though.

Reply:
It was enough to show up on @xiangfan-ms's radar. Clarity will be dramatically improved when we can `if constexpr`ize this, hopefully soon.